packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -364,6 +372,33 @@
       <artifactId>curator-recipes</artifactId>
       <version>${zk-curator.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-flink_${scala.binary.version}</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-flink-client</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
We may need a separate profile for flink or a separate bundle for hudi-flink-utilities

Reply:
@danny0405  we discussed on the mailing regarding this and decided to go ahead with using a single utilities bundle for Flink and spark. 

cc @vinothchandar